docs/configuration.md

```diff
@@ -56,6 +56,28 @@ metadata:
   
 In the example above the PVC called **myclaim** will be mounted to **/mount/path/from/annotation** directory
 
+#### Object annotation for mounting a volume to specific container(s)
+
+Using **rhdh.redhat.com/containers** annotation it is possible to define the containers where **PersistentVolumeClaim** object will be mounted.
```

Comment:
This implies that the mount path will be the same in all the containers listed here, right?
What if I want to mount the PVC under specific paths for each container? IMO, we should handle this.
Thinking out loud, but how about extending the behavior of the `rhdh.redhat.com/mount-path` annotation above to include the container names, e.g.: `rhdh.redhat.com/mount-path: 'install-dynamic-plugins:/mount/path1,backstage-backend:/mount/path2'`?

Reply:
Thought about it (and discussed with Tomas)... Can not see the case when it should be really important for *default* configuration. We can (and will when needed) make it more flexible for CR but I'd really avoid scripting in annotations unless really unavoidable.